bash_scripts/cron.jobs

```diff
@@ -6,7 +6,8 @@ LOG_PATH  = /home/jaleazo/projects/Spotify-2.0/logs
 
 * * * * *  $PROJ_PATH/Proj.sh Playback_Tracker  >  $LOG_PATH/Playback-Tracker.log    2>&1    # Every Minute
```

Comment:
Is this `>` instead of `>>` intentional? This seems to overwrite `Playback-Tracker.log` every minute, whereas all these other cronjobs append to the existing file

Reply:
Yep, since Playback-Tracker runs every minute and also technically runs the script internally 4 times a second we really don't want this to compound if things start breaking. This log is more if it is "currently" broken.